app/src/main/java/com/besga/jonander/avisaaane/StarterApplication.java

```diff
@@ -17,18 +20,19 @@ public void onCreate() {
         Properties prop = new Properties();
         try {
             prop.load(getAssets().open("app.properties"));
-
-
         } catch (IOException ex) {
-            ex.printStackTrace();
+            Log.e(TAG, "File app.properties with ApplicationID and ClientKey not found", ex);
+            Intent homeIntent = new Intent(Intent.ACTION_MAIN);
```

Comment:
Yep, the idea is that instead of giving no warnings nor anything, it will print the stacktrace of the error and leave. The best would be to create a dialog saying XXX but I was lazy...